migrations/20230922123931-create-game-for-user.cjs

```diff
@@ -27,8 +41,29 @@ module.exports = {
         type: Sequelize.DATE
       }
     });
+    await queryInterface.addConstraint('GameForUsers', {
+      type: 'foreign key',
+      fields: ['user_id'],
+      references: {
+        table: 'users',
+        field: 'id',
+      },
+      onDelete: 'CASCADE',
+      onUpdate: 'CASCADE',
+    });
+
+    await queryInterface.addConstraint('GameForUsers', {
+      type: 'foreign key',
+      fields: ['game_id'],
+      references: {
+        table: 'Games',
+        field: 'id',
+      },
+      onDelete: 'CASCADE',
+      onUpdate: 'CASCADE',
+    });
   },
   async down(queryInterface, Sequelize) {
     await queryInterface.dropTable('GameForUsers');
```

Comment:
Do you realise that you have changed the table name to GameForUsers and hence the SQL queries you make in your controllers such as the join query I made will fail because they reference the previous table `usergame` instead?